com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/NodeUIDescriptor.cs

```diff
@@ -30,7 +31,8 @@ namespace UnityEditor.ShaderGraph.Defs
             string displayName = null,
             bool hasPreview = true, // By default we assume all nodes should have previews,
             Dictionary<string, string> selectableFunctions = null,
-            ParameterUIDescriptor[] parameters = null
+            ParameterUIDescriptor[] parameters = null,
+            string functionSelectorLabel = ""
```

Comment:
I think there is maybe a future where the function selector is rationalized as a parameter-- not important here, but maybe something to think about in the future.